_data/patterns/race.yml

```diff
@@ -0,0 +1,88 @@
+legend: |
+  What is your race and/or ethnicity?
+hint: |
+  Select all that apply
+hint_input: |
+  and enter additional details in the spaces below
```

Comment:
It looks like the pipe and new line are causing some of the whitespace issues in the component preview:

![image](https://github.com/uswds/uswds-site/assets/25211650/6852bba1-4b50-4d0d-b0b9-435fbbd597b0)

Removing them and rebuilding resolves some them:

![image](https://github.com/uswds/uswds-site/assets/25211650/59eac80b-e45b-4662-929b-0bc8b98eaec5)

Would we be ok to remove these?

```suggestion
legend: What is your race and/or ethnicity?
hint: Select all that apply
hint_input: and enter additional details in the spaces below
```

Reply:
Good fix. Updated in [a0efd7b](https://github.com/uswds/uswds-site/pull/2657/commits/a0efd7b9af54fc75b01a168fbc6f30692d0af614)!